test/framework/src/main/java/org/elasticsearch/test/VersionUtils.java

```diff
@@ -149,9 +147,13 @@
      * Get the released version before {@code version}.
      */
     public static Version getPreviousVersion(Version version) {
-        int index = RELEASED_VERSIONS.indexOf(version);
-        assert index > 0;
-        return RELEASED_VERSIONS.get(index - 1);
+        for (int i = RELEASED_VERSIONS.size() - 1; i >= 0; i--) {
```

Comment:
This change is required for `getPreviousVersion(CURRENT)` to work. We do that fairly frequently.